homeassistant/components/cast/media_player.py

```diff
@@ -958,4 +962,4 @@ def _handle_signal_show_view(
             self._hass_cast_controller = controller
             self._chromecast.register_handler(controller)
 
-        self._hass_cast_controller.show_lovelace_view(view_path)
+        self._hass_cast_controller.show_lovelace_view(view_path, url_path)
```

Comment:
Do we need to upgrade pychromecast for this or did we already do?

Reply:
We already did